ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -395,6 +395,11 @@ esac
 
 sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
 
+if [ "${MASTER_SCHEDULABLE}" = "true" ]; then
```

Comment:
Needs to be a separate step which creates a scheduler config separately. The lines below will copy it to manifest/ file and override default settings

Reply:
Looks like manifest is [created](https://github.com/openshift/release/blob/bef7089a4f958a65f2737d084ef11cfbebf0cd6a/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L383) and [consumed](https://github.com/openshift/release/blob/bef7089a4f958a65f2737d084ef11cfbebf0cd6a/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L477) in this same step, so a separate step may not work

Reply:
The additional manifest should override default setting, so it needs to be placed after the default one in lexicograpghical order (i.e. `99-override-masters-schedulable.yaml`)

Reply:
This step comes from doc https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configure-network-components-to-run-on-the-control-plane_ipi-install-installation-workflow, it's better we keep in consistency.

Reply:
@vrutkovs Do you happen to know if `cluster-scheduler-02-config.yml` is varying version by versio? If it is, I have some concern of the overriding of manifest would cause some compatibility issues between versions?

Reply:
I don't think it has ever changed. My main concern with this change is that we're putting too many things in ipi-install-install, instead we should add separate steps

Reply:
@vrutkovs Get your point and added a separate step, thanks.

Reply:
@vrutkovs [Rehease job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/36786/rehearse-36786-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-ipi-remote-worker/1632948747044392960) failed, seems like can not override cluster-scheduler-02-config.yml, do you have any suggesstions?
`level=fatal msg=failed to fetch Metadata: failed to fetch dependency of "Metadata": failed to fetch dependency of "Bootstrap Ignition Config": failed to generate asset "CVO Ignore": multiple manifests for group config.openshift.io kind Scheduler namespace  name cluster: manifests/99-override-masters-schedulable.yaml, manifests/cluster-scheduler-02-config.yml
Installer exit with code 1`